src/ipLoadBalancing/farm/ipLoadBalancing-farm-server.lexi.service.js

```diff
@@ -0,0 +1,41 @@
+"use strict";
```

Comment:
You should encapsulate this declaration, else, you declare `"use strict"` for all code :/

Reply:
What, you mean that we don't use IIFEs? 😳 :troll:

Reply:
done ;)